src/models/node.js

```diff
@@ -1329,6 +1321,40 @@ const Node = {
   },
 
   /**
+   * Split a node by `path` after 'count' children.
+   * Does not work on Text nodes. Use `Node.splitNode` to split text nodes as well.
+   *
+   * @param {Array} path
+   * @param {Number} count
+   * @return {Node}
+   */
+
+  splitNodeAfter(path, count) {
```

Comment:
What if the original `joinNode` operation "zippered" all the way down to the text nodes? That seems like a reasonable thing for it to do when joining, which would then make splitting simple?